build/source/engine/read_force.f90

```diff
@@ -126,7 +127,8 @@ subroutine read_force(istep,iHRU,err,message)
                   refTime%var(iLookTIME%im),              & ! input  = month
                   refTime%var(iLookTIME%id),              & ! input  = day
                   refTime%var(iLookTIME%ih),              & ! input  = hour
-                  refTime%var(iLookTIME%imin),dsec,       & ! input  = minute/second
+                  refTime%var(iLookTIME%imin),            & ! output = minute
+	          refTime%var(iLookTIME%isec),            & ! output = second
```

Comment:
Split out the time step changes from the GRU changes (or explain why they are related).

Fix indenting (below as well)